server/src/main/java/org/opensearch/search/aggregations/bucket/histogram/HistogramAggregationBuilder.java

```diff
@@ -227,12 +227,12 @@ public HistogramAggregationBuilder offset(double offset) {
 
     /** Get the current minimum bound that is set on this builder. */
     public double minBound() {
```

Comment:
@GeorgeAp there is obviously an issue with these methods handling `null`s but I have not find any usages of these methods in the codebase, I am curious what flow you run into NPEs?

Reply:
@reta the flow is mainly, we create an aggregator using this builder but choose to use the default values i.e. `POSITIVE_INFINITY` and `NEGATIVE_INFINITY` but check to ensure that is this is the case. To check, the methods are called and since they are not set, they throw NPE.